_posts/2015-02-09-create-your-application.markdown

```diff
@@ -118,7 +116,7 @@ Now that we have started our Rails application, we need to install GeoBlacklight
      {: .flash-notice}
 
      ```sh
-     $ rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
+     $ DISABLE_SPRING=1 rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/main/template.rb
```

Comment:
We might want to remind users to cd into the new directory before running the next rake task?

Reply:
Good point! I've pushed up that change.